### DISCUSSION TITLE
List of supported AVD

### DISCUSSION THREAD

**Discussion by HamzaNiazi1995** [ReactiveCircus/android-emulator-runner#407] [Q&A]
I am using this in  a CI for github actions is there a list of supported devices related to this.
Currently it is this
name: Start Emulator, Verify Emulator, and Run Android Tests
        uses: reactivecircus/android-emulator-runner@v2
        with:
          api-level: 29
          profile: Galaxy Nexus
          disable-animations: false
          
          I want it on  pixel 8 would that be possible

**Comment by jprodriguez-sw** [comment]
You should be able to see the available profiles by running `avdmanager list device`, I don't see `Pixel 8` in my case.
